src/ServicePulse.Host/vue/src/composables/autoRefresh.ts

```diff
@@ -3,7 +3,7 @@ export default function useAutoRefresh(refreshAction: () => Promise<void>, timeo
 
   function stopTimer() {
     if (refreshInterval !== null) {
-      window.clearTimeout(refreshInterval);
+      window?.clearTimeout(refreshInterval);
```

Comment:
@PhilBastian how about now?

Reply:
for consistency do you want to put the same null propagation on the setTimeout call also? It shouldn't change the type of the refreshInterval handle, since it can already be null